docs/content/TreeView.mdx

```diff
@@ -19,35 +19,35 @@ description: A hierarchical list of items where nested items can be expanded and
         </TreeView.LeadingVisual>
         src
         <TreeView.SubTree>
-          <TreeView.LinkItem id="src/Avatar.tsx" href="#">
+          <TreeView.Item id="src/Avatar.tsx">
```

Comment:
How do you feel about replacing the `href` prop in these examples with an `onSelect` callback that logs to the console? I think that might make the docs a bit more clear.

```suggestion
          <TreeView.Item id="src/Avatar.tsx" onSelect={() => console.log('src/Avatar.tsx')}>
```

Reply:
Good idea!